### PR TITLE
myGuide - Confirm Tour Screen

### PR DESCRIPTION
**Pull request by MattArwenLangham**
This PR completes confirm Tour ID screen - https://trello.com/c/e7X5eaSy/51-fe-jointour-tour-id-confirmation-screen

**This branch adds**:
- A confirmation screen for the tour ID, displaying the inputted tour ID.
- Passing the tour ID through as a prop to allow for future implementation of passing it to the backend server.

_Once the API is set up this will implement more details such as the tour picture/tour name/tour description etc._

### CODE REVIEW

**Review by callumu21 — APPROVED** [review]
Nice! Loving the transitional loading message before the map pops in.

File: src/components/user/Map.js [hunk 1]
```diff
@@ -24,37 +24,31 @@ const Map = () => {
...
   }, [])
-
-  let text = 'Tour loading...'
   
-  if (errorMsg) {
-    text = errorMsg
-  } else if (latitude || longitude) {
-    text = latitude, longitude
-  }
+  if (errorMsg) return <Text>{erroMsg}</Text>
```
Comment (BrennB99) [inline src/components/user/Map.js]: errorMsg is spelt incorrectly

**Review by BrennB99 — APPROVED** [review]
Once the spelling mistake is fixed it works perfectly so feel free to merge

File: src/components/user/Map.js (hunk 1 above)
Comment (MattArwenLangham) [reply src/components/user/Map.js]: Phenomenal catch! Thanks Ben, already changed, and pushed so will merge now :)

**Pull request merged**